airbyte-ci/connectors/pipelines/pyproject.toml

```diff
@@ -40,6 +40,7 @@ poethepoet = "^0.24.2"
 
 [tool.poetry.scripts]
 airbyte-ci = "pipelines.cli.airbyte_ci:airbyte_ci"
```

Comment:
I left this in for now as I have a hope that the auto upgrade feature will install the binary automatically for people